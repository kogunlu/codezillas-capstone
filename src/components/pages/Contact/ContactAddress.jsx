import React from "react";
import ContactAdressImg from "./images/ContactAdressImg.png";


function ContactAddress () {
   return ( 
   <div>
    <div className="relative mt-28 ml-15 m-10">
        <img className="h-56 w-96 md:h-auto md:w-auto "src={ContactAdressImg} alt="adress" /> 
      <span className="absolute top-3 left-3 w-4/6 pl-5 pt-3">
              <p className="font-medium text-xs md:text-lg text-start mb-1">
                Find Us At: 
              </p>
             <div className="text-gray-500 text-xs md:text-lg">
              <p>Nergiz Plaza</p>
              <p>3rd Floor</p>
              <p>Bakhtiyari Street 40m</p>
              <p>Erbil, Iraq</p>
              <p>44001</p>
              </div>
            </span>
            </div>
            </div>
      
   )
}
export default ContactAddress ;