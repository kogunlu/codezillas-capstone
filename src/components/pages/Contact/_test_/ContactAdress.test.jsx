import { BrowserRouter } from "react-router-dom"
import renderer from "react-test-renderer";
import ContactAddress from "../ContactAddress"


it('Renders correctly for ContactAddress Component', () => {
    const tree = renderer
    .create(
    <BrowserRouter>
    <ContactAddress />
    </BrowserRouter>)
    .toJSON();
    expect(tree).toMatchSnapshot();
  }); 
