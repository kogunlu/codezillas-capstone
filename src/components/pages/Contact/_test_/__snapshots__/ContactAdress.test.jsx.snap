// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly for ContactAddress Component 1`] = `
<div>
  <div
    className="relative mt-28 ml-15 m-10"
  >
    <img
      alt="adress"
      className="h-56 w-96 md:h-auto md:w-auto "
      src="ContactAdressImg.png"
    />
    <span
      className="absolute top-3 left-3 w-4/6 pl-5 pt-3"
    >
      <p
        className="font-medium text-xs md:text-lg text-start mb-1"
      >
        Find Us At:
      </p>
      <div
        className="text-gray-500 text-xs md:text-lg"
      >
        <p>
          Nergiz Plaza
        </p>
        <p>
          3rd Floor
        </p>
        <p>
          Bakhtiyari Street 40m
        </p>
        <p>
          Erbil, Iraq
        </p>
        <p>
          44001
        </p>
      </div>
    </span>
  </div>
</div>
`;
